contributing/web_platform_tests.md

```diff
@@ -125,9 +125,14 @@ filters after a `--`:
 
 ```shell
 cd test_util/wpt/
-# Rebase to retain our modifications
-git rebase origin/master
-git push denoland
+git fetch origin
+git checkout origin/master
```

Comment:
```suggestion
git checkout origin/epochs/daily
```

Reply:
Why not update to master? I don't think this would interact with the `wpt_epoch` workflow, even in the extremely improbable case in which the wpt submodule was checked out to a commit more recent than `epochs/daily` by the time the workflow runs.